.env.sample.l3

```diff
@@ -23,7 +23,7 @@ DISABLE_P2P_SYNC=false
 
 ############################### REQUIRED #####################################
 # L2 RPC endpoints (you will need to run a fully synced L2 node to start a L3 node)
-# If you are using a local Taiko L2 node, you can refer to it as "host.docker.internal" or use the local IP address
+# If you are using a local Taiko L2 node, you can refer to it as "host.docker.internal:8546", "host.docker.internal:8547" port is different from the .env since it is accessing internally.
```

Comment:
does this need to be http://host.docker.internal:8546 and ws://host.docker.internal:8547? this did not work for me without adding an extra host to my docker-compose.yml but seems like it works for a lot of people.

it would also be nice to retain the option of using the private local IP address, in case it also doesn't work for others.